node/src/index-send-eth-to-payid.js

```diff
@@ -0,0 +1,143 @@
+const axios = require('axios')
+const EthereumTx = require('ethereumjs-tx').Transaction
+const open = require('open')
+const Web3 = require('web3')
+const { PayIDClient } = require('xpring-js')
+
+/**
+ * The address you are sending from
+ * @type {String}
+ */
+const fromAddress = 'ADD YOUR ADDRESS HERE' // 0xD0544A81F0962F7ff0d08Eaf1E5B226DAD331316
```

Comment:
Any reason to not make this a test-net demo and to just provide these values up front?

Reply:
Added working values